storage/migrations/18_evm_nfts.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE chain.evm_nfts (
+    runtime runtime NOT NULL,
+    token_address oasis_addr NOT NULL,
+    nft_id uint_numeric NOT NULL,
+    PRIMARY KEY (runtime, token_address, nft_id),
+    metadata_uri TEXT,
+    metadata_accessed TIMESTAMP,
+    name TEXT,
+    description TEXT,
+    image TEXT
```

Comment:
just the url

can we put that image fetch in the frontend at first?

Reply:
Yeah IIRC @lukaw3d said there's no csrf in place because the explorer is read-only. So for the short term, letting them fetch directly from untrusted sources should be fine.

Reply:
Yep yep, we don't have CSP yet
(also no CSRF, but that's different)

Reply:
resolving this to focus on active discussions :skull: